commands/entity-store.mjs

```diff
@@ -113,7 +114,10 @@ const assignAssetCriticalityToEntities = async (entities, field) => {
  * Then Generate events, assign asset criticality, create rule and enable risk engine
  * @param {*} param0
  */
-export const generateEntityStore = async ({ users = 10, hosts = 10 }) => {
+export const generateEntityStore = async ({ users = 10, hosts = 10, options}) => {
+  if(options.includes(ENTITY_STORE_OPTIONS.seed)) {
+    faker.seed(12345);
```

Comment:
yeah ok, I just added a random seed, which gets printed so people can store it, and a config option for passing in a seed.  I can adapt it if we merge this first